widgetastic_manageiq.py

```diff
@@ -878,3 +878,68 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class ItemsToolBarViewSelector(View):
+    """ represents toolbar's view selector control
+        it is present on pages with items like Infra or Cloud Providers pages
+
+    .. code-block:: python
+
+        @View.nested
+        class view_selector(ItemsToolBarViewSelector):  # NOQA
+
+        some_view.view_selector.select('Tile View')
+        some_view.view_selector.selected
+    """
+    ROOT = './/div[contains(@class, "toolbar-pf-view-selector")]'
+    grid_button = Button(title='Grid View')
+    tile_button = Button(title='Tile View')
+    list_button = Button(title='List View')
+
+    @property
+    def _view_buttons(self):
+        return self.grid_button, self.tile_button, self.list_button
```

Comment:
I have an idea :)

```python
# Since this is only used as an iterable
def _view_buttons(self):
    yield self.grid_button
    yield self.tile_button
    yield self.list_button
```

:)